third_party_libraries/kml.js

```diff
@@ -212,9 +213,10 @@ L.Util.extend(L.KML, {
 		el = xml.getElementsByTagName('Placemark');
 		for (var j = 0; j < el.length; j++) {
 			if (!this._check_folder(el[j], xml)) { continue; }
-			l = this.parsePlacemark(el[j], xml, style);
-
-			if (l) { layers.push(l); }
+			var parsedLayers = this.parsePlacemark(el[j], xml, style);
```

Comment:
Maybe it's my lack of morning coffee, but this set of lines looks similar to [these](https://github.com/TourTrak/devcycle-mobile-app/pull/21/files#diff-258bab95ef85f982486c9057a0e410eaR103).  Are they related in some way?